src/libraries/System.Security.Cryptography.Cng/tests/PropertyTests.cs

```diff
@@ -160,5 +160,33 @@ public static void NullValueRoundtrip()
             CngProperty property = new CngProperty("banana", null, CngPropertyOptions.CustomProperty);
             Assert.Null(property.GetValue());
         }
+
+        [Fact]
+        public static void TestConstructorSpan()
+        {
+            var name = "dotnet-test";
+            ReadOnlySpan<byte> value = new byte[12];
+            value[5] = 1;
+            value[6] = 2;
+            value[7] = 3;
```

Comment:
This is a `ReadOnlySpan` so it can’t be written to.
```suggestion
            ReadOnlySpan<byte> value = [1, 2, 3, 4, 5];
```